__tests__/app.test.js

```diff
@@ -29,3 +30,13 @@ describe("GET /api/topics", () => {
 			});
 	});
 });
+describe("GET /api", () => {
+	it("GET 200 - responds with a JSON object detailing all available endpoints", () => {
+		return request(app)
+			.get("/api")
+			.expect(200)
+			.then(({ body: { endpoints } }) => {
+				expect(endpoints).toEqual(endpointsData);
+			});
+	});
```

Comment:
Lovely work here! I think it might be nice to add another test that checks they have the correct key-value pairs as well? :)